RELEASING.md

```diff
@@ -12,16 +14,32 @@ This document explains the release strategy for artifacts in this organization.
 
 ## Branching
 
-### Release Branching
+Projects create a new branch when they need to start working on 2 separate versions of the product, with the `main` branch being the furthermost release. 
 
-Given the current major release of 1.0, projects in this organization maintain the following active branches.
+### OpenSearch Branching
 
-* **main**: The next _major_ release. This is the branch where all merges take place and code moves fast.
-* **1.x**: The next _minor_ release. Once a change is merged into `main`, decide whether to backport it to `1.x`.
-* **1.0**: The _current_ release. In between minor releases, only hotfixes (e.g. security) are backported to `1.0`.
+[OpenSearch](https://github.com/opensearch-project/OpenSearch) typically tracks 3 releases in parallel. For example, given the last major release of 1.0, OpenSearch in this organization maintains the following active branches.
+
+* **main**: The _next major_ release, currently 2.0. This is the branch where all merges take place and code moves fast.
+* **1.x**: The _next minor_ release, currently 1.1. Once a change is merged into `main`, decide whether to backport it to `1.x`.
+* **1.0**: The _current_ release, currently 1.0. In between minor releases, only hotfixes (e.g. security) are backported to `1.0`. The next release out of this branch will be 1.0.1.
 
 Label PRs with the next major version label (e.g. `2.0.0`) and merge changes into `main`. Label PRs that you believe need to be backported as `1.x` and `1.0`. Backport PRs by checking out the versioned branch, cherry-pick changes and open a PR against each target backport branch.
 
+### Plugin Branching
+
+Plugins, such as [job-scheduler](https://github.com/opensearch-project/job-scheduler) aren't as active as OpenSearch, and typically track 2 releases in parallel instead of 3. For example, given the last major release of 1.0, job-scheduler maintains the following active branches.
```

Comment:
I understand why we'd like to reduce it to 2 branches. 
We should communicate to plugins and remove 1.x branches if they've already created.

Reply:
I actually don't think a 1.x branch is a bad idea. It will make things easier to merge. I'll update the text.

Reply:
Can we explicitly add that 1,x is optional for plugins?

Reply:
But it's not. What's optional is that you _work_ on the 1.x branch to do active development. Which is explained in the example right below this paragraph.

Reply:
You're right. See my update in https://github.com/opensearch-project/.github/pull/33/commits/eb83b9de6b540d5d92564a22450545789212d8a0